Makefile

```diff
@@ -45,5 +51,11 @@ devtools:
 
 #? help: Get more info on make commands.
 help: Makefile
-	@echo " Choose a command run in go-ethereum:"
+	@echo ''
+	@echo 'Usage:'
+	@echo '  make [target]'
+	@echo ''
+	@echo 'Targets:'
 	@sed -n 's/^#?//p' $< | column -t -s ':' |  sort | sed -e 's/^/ /'
+
+.DEFAULT_GOAL := help
```

Comment:
help should not be the default. the default is to build.

Reply:
Fixed in [c8e205a](https://github.com/ethereum/go-ethereum/pull/29777/commits/c8e205a5e46f51e1e66bd09e9308fc1fb3cbb731).